lib/manager.js

```diff
@@ -126,13 +126,17 @@ inherits(Manager, EventEmitter)
  * @private
  */
 Manager.prototype.open = function (uri, opts, fn) {
-  MongoClient.connect(uri, opts, function (err, db) {
-    if (err || !db) {
+  opts.useNewUrlParser = true
```

Comment:
`opts` are optional so we need to make sure it's defined before adding a field to it.

Additionally, it'd be best to only set this option when it's not specified already (eg. a user might want to not use the new parser

Reply:
You're right.

I moved the `useNewUrlParser` option handling on `Manager`, where there is already a check `opts = opts || {}`.

I've added some tests and refactor to make the user override the default value (`true`) when needed.